pkgs/tools/security/keybase/kbfs.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, buildGoPackage, fetchFromGitHub, keybase }:
+
+buildGoPackage rec {
+  pname = "kbfs";
+  # must have the same version as the keybase package
+  version = keybase.version;
+
+  goPackagePath = "github.com/keybase/client";
+  subPackages = [ "go/kbfs/kbfsfuse" "go/kbfs/kbfsgit/git-remote-keybase" ];
+
+  dontRenameImports = true;
+
+  src = keybase.src;
```

```suggestion
  inherit (keybase) src version;
```

And also remove version assignment above.